ipv8/REST/overlays_endpoint.py

```diff
@@ -30,3 +33,90 @@ def get_overlays(self):
 
     def render_GET(self, request):
         return json.dumps({"overlays": self.get_overlays()})
+
+
+class OverlayStatisticsEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handing all requests regarding the statistics of overlays.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self.session = session
+        self.statistics_supported = isinstance(self.session.endpoint, StatisticsEndpoint)
+
+    def get_statistics(self):
+        overlay_stats = []
+        for overlay in self.session.overlays:
+            statistics = self.session.endpoint.get_statistics(overlay.get_prefix()) if self.statistics_supported else {}
+            overlay_stats.append({
+                overlay.__class__.__name__: self.statistics_by_name(statistics, overlay)
+            })
+        return overlay_stats
+
+    def statistics_by_name(self, statistics, overlay):
+        named_statistics = {}
+        for message_id, network_stats in statistics.items():
+            mapped_name = str(message_id)+":"+overlay.decode_map[chr(message_id)].__name__
```

Comment:
Instead of `chr(message_id)` please use this for Python 3 support:

https://github.com/Tribler/py-ipv8/blob/02add669e7b768169b8e8d2f8555cba2e9b60c50/ipv8/deprecated/community.py#L265